db/sql/create-timeEntry-and-timePeriodType-table.sql

```diff
@@ -0,0 +1,32 @@
+
+CREATE TABLE timecard.time_period_type (
+    id VARCHAR(36)  PRIMARY KEY,
+    tenant_id VARCHAR(36) NOT NULL,
+    name VARCHAR(50) NOT NULL
+);
+
+INSERT INTO timecard.time_period_type (id,tenant_id,"name") VALUES
+	 ('00000000-0000-0000-0000-000000000001','00000000-0000-0000-0000-000000000000','Shift'),
+	 ('00000000-0000-0000-0000-000000000002','00000000-0000-0000-0000-000000000000','Scheduled rest day'),
+	 ('00000000-0000-0000-0000-000000000003','00000000-0000-0000-0000-000000000000','Non-working day'),
+	 ('00000000-0000-0000-0000-000000000004','00000000-0000-0000-0000-000000000000','On call'),
+	 ('00000000-0000-0000-0000-000000000005','00000000-0000-0000-0000-000000000000','Absence'),
+	 ('00000000-0000-0000-0000-000000000006','00000000-0000-0000-0000-000000000000','Training'),
+	 ('00000000-0000-0000-0000-000000000007','00000000-0000-0000-0000-000000000000','Overtime');
+
+CREATE TABLE timecard.time_entry (
+    id VARCHAR(36)  PRIMARY KEY,
+    tenant_id VARCHAR(36) NOT NULL,
+    version INT DEFAULT NULL,
+    owner_id INT NOT null,
+    time_period_type_id VARCHAR(36) NOT NULL,
+    shift_type VARCHAR(15) DEFAULT NULL,
+    actual_start_time TIMESTAMP NOT NULL,
+    actual_end_time TIMESTAMP DEFAULT NULL,
+    planned_start_time TIMESTAMP DEFAULT NULL,
+    planned_end_time TIMESTAMP DEFAULT NULL,
+    created_at TIMESTAMP NOT NULL,
+    updated_at TIMESTAMP NOT NULL,
+    deleted BOOLEAN DEFAULT NULL,
```

Comment:
I'm assuming the created_at, updated_at, deleted are for our own general auditing purposes. If so would we also want a deletedAt?

Reply:
No, updated_at should be updated whatever the activity you do

Reply:
ok so updated_at would also account for the time it was deleted, sounds good.